audformat/core/utils.py

```diff
@@ -376,6 +360,43 @@ def join_labels(
     return joined_labels
 
 
+def join_schemes(
+        dbs: typing.Sequence[Database],
+        scheme_id: str,
+):
+    r"""Join and update scheme of two databases.
```

Comment:
Could you please update the docstring as we can now combine the scheme of more than two databases.

Reply:
done